industrial_ci/src/tests/source_tests.sh

```diff
@@ -25,11 +25,18 @@
 ici_require_run_in_docker # this script must be run in docker
 
 #Define some verbose env vars
+#verbose build tests
```

Comment:
It's just "verbose build", not just tests

Reply:
done